stable/hackmd/templates/deployment.yaml

```diff
@@ -38,7 +42,9 @@ spec:
               port: 3000
             initialDelaySeconds: 30
           env:
-            - name: HMD_DB_PASSWORD
+            - name: NODE_ENV
+              value: "production"
```

Comment:
nit. NODE_ENV=production is already set by https://github.com/hackmdio/codimd-container/blob/master/debian/Dockerfile#L10 and https://github.com/hackmdio/codimd-container/blob/master/alpine/Dockerfile#L9

Reply:
removed `NODE_ENV`